libminifi/include/ResourceClaim.h

```diff
@@ -104,9 +96,8 @@ class ResourceClaim : public std::enable_shared_from_this<ResourceClaim> {
   }
 
  protected:
-  std::atomic<bool> deleted_;
   // Full path to the content
-  std::string _contentFullPath;
+  const Path _contentFullPath;
```

Comment:
I'm not sure `const` is a good idea here. It disables copy/move assignment, but not construction. 

More details: https://abseil.io/tips/177
I tend to generalize these guidelines to "generally avoid `const` or reference non-static data members".

Reply:
a few lines below this we have the copy ctor and the copy assignment declared but undefined and the comment:

> // Prevent default copy constructor and assignment operation
// Only support pass by reference or pointer

so I think based on this, ResourceClaim was intended to be a `immutable type` of which, the linked resource claims the following:

> There is one useful-but-unusual design that may mandate const members: intentionally immutable types. Instances of such a type are immutable after construction: no mutating methods, no assignment operators. 

of course this might not mean anything, I can remove the `const`, should I do it?

Reply:
In this case I leave the decision to your taste.